tools/wpt/browser.py

```diff
@@ -644,10 +643,21 @@ def find_webdriver(self, channel=None):
     def install_webdriver(self, dest=None, channel=None):
         raise NotImplementedError
 
-    def version(self, binary):
+    def version(self, binary=None, webdriver_binary=None):
+        if webdriver_binary is not None:
+            try:
+                output = call(webdriver_binary, "--version").strip()
+                # TODO: explain
```

Comment:
What needs explaining is why this string begins with 'Included with '.

But more importantly, are there any requirements on the shape of this string? See https://github.com/web-platform-tests/results-collection/issues/617 for the ouput and what it's presented like in UI.